CMakeLists.txt

```diff
@@ -0,0 +1,48 @@
+project(OpenDirectory)
+
+set(DYLIB_COMPAT_VERSION "1.0.0")
+set(DYLIB_CURRENT_VERSION "1.0.0")
+
+include_directories(${CMAKE_CURRENT_SOURCE_DIR}/Core/include)
+
+add_framework(CFOpenDirectory
+    FAT
+    CURRENT_VERSION
+    VERSION "A"
+    PARENT "OpenDirectory"
```

Comment:
@ahyattdev This was what I was talking about on discord. I was wondering what the advantage of `InstallSymlink` is compared to using `PARENT`.

Reply:
Hi Tommy, the PARENT part tells `add_darling_framework` that this framework needs to be installed inside of another framework.

Due to limitations of CMake, we can't install the symlink automatically for nested frameworks and we must do it outside of that function.

There isn't an advantage over one of the other, they're both necessary.

Reply:
Okay, I'll look into adding the `InstallSymlink` function after my midterms.

Reply:
@CuriousTommy right on dude

Reply:
@ahyattdev So even if `CFOpenDirectory` only exists in `OpenDirectory` and nowhere else, we are still required to use InstallSymlink.

Also, where does `../../../../` lead to (I am refering to the [ApplicationServices/CMakeLists.txt](https://github.com/darlinghq/darling/blob/3aea9eb1700fc8ae5981b1c31c526a4927720161/src/ApplicationServices/CMakeLists.txt#L49))